ci-team.members.txt

```diff
@@ -10,3 +10,4 @@ redhat-team@kubeflow.org
 wibuch@microsoft.com
 yangpa@google.com
 holden.karau@gmail.com
+kam.d.kasravi@intel.com
```

Comment:
Kam, looks like most of the file was alphabetical in order except for holden. Can I request that you move your entry and holden's entry to appropriate order.